fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/RebuildIT.java

```diff
@@ -57,28 +54,20 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RebuildIT.class);
 
-    private static String origStorageRootDir;
-    private static String origWorkDir;
+    private static final String OCFL_ROOT_PROP = "fcrepo.ocfl.root";
```

Comment:
Would it make sense to define this constant statically on OcflPropsConfig.java?

Reply:
done